components/pages/explorer/RepoTable/index.tsx

```diff
@@ -408,7 +409,7 @@ const RepoTable = (props: PageContentProps): ReactElement => {
           allowTSVExport={false}
           showFilterInput={false}
           columnDropdownText={'Columns'}
-          customHeaderContent={DownloadButton}
+          customHeaderContent={NEXT_PUBLIC_ENABLE_DOWNLOADS && DownloadButton}
```

Comment:
only display the `download` button if enabled

this could also be useful for `singularity` maintenance.